projects/replicated-ship/schema.json

```diff
@@ -37,16 +58,51 @@
                   "source"
                 ]
               },
+              "github": {
+                "type": "object",
+                "properties": {
+                  "description": {
+                    "type": "string"
+                  },
+                  "dest": {
+                    "type": "string"
+                  },
+                  "mode": {
+                    "type": "integer"
+                  },
+                  "path": {
+                    "type": "string"
+                  },
+                  "private": {
+                    "type": "boolean"
+                  },
+                  "ref": {
+                    "type": "string"
+                  },
+                  "repo": {
+                    "type": "string"
+                  }
+                },
+                "required": [
+                  "dest",
+                  "mode",
+                  "description",
```

Comment:
did you mean to make `description` be a required field?

Reply:
same for `mode` -- might not make sense if we are going to inherit the file mode from the git repo